### PR TITLE
Fix dropdown menu position & update reject error codes

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This PR adjusts HTML and `daisyUI` classes so that the user menu appears below the gravatar. Errors recently added to the API docs have also been added to the transaction rejection modal options and `REJECT_CODES` object.

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/30465856?key=299667c81c43b5a6c177c7bda926fd3a

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/static/js/constants.js [hunk 1]
```diff
@@ -327,19 +327,24 @@ export const naturalPersonNameTypeArray = Object.entries(IDENTIFIER_TYPE)
...
-  UNKNOWN_IDENTITY: "Unknown Identity",
-  UNKNOWN_ORIGINATOR: "Unknown Originator",
-  UNKNOWN_BENEFICIARY: "Unknown Beneficiary",
-  UNSUPPORTED_CURRENCY: "Unsupported Currency",
-  EXCEEDED_TRADING_VOLUME: "Exceeded Trading Volume",
-  COMPLIANCE_CHECK_FAIL: "Compliance Check Fail",
-  NO_COMPLIANCE: "No Compliance",
-  HIGH_RISK: "High Risk",
-  OUT_OF_NETWORK: "Out of Network",
+  REJECTED: "Default rejection - no specified reason for rejecting the transaction",
```
Comment (daniellemaxwell) [inline pkg/web/static/js/constants.js:330]: This was updated to be more in line with what's listed in the API docs. The values would be displayed in the error envelope or at the top of the repair form.

File: pkg/web/templates/components/transaction_rejection.html [hunk 2]
```diff
@@ -28,6 +29,10 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
             <option value="UNPARSEABLE_TRANSACTION">Unable to parse transaction data record</option>
             <option value="MISSING_FIELDS">Missing required fields in transaction data</option>
             <option value="INCOMPLETE_IDENTITY">Identity record is not complete</option>
+            <option value="VALIDATION_ERROR">There was an error validating a field in the transaction data</option>
+            <option value="COMPLIANCE_PERIOD_EXCEEDED">The review period has exceeded the required compliance timeline</option>
+            <option value="CANCELED">Cancel the ongoing TRISA exchange and do not send funds</option>
+            <option value="CANCEL_TRANSACTION">Cancel the ongoing TRISA exchange and do not send funds (Alias: CANCELED)</option>
```
Comment (bbengfort) [inline pkg/web/templates/components/transaction_rejection.html:35]:
```suggestion
```

I don't think the alias is needed in the dropdown.

File: pkg/web/templates/components/transaction_rejection.html [hunk 3]
```diff
@@ -14,6 +14,7 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
             <select id="transaction-rejection" name="code"
             class="!bg-gray-50 border !border-gray-300 !text-gray-900 rounded-lg focus:!ring-blue-500 focus:!border-blue-500 block w-full p-2.5">
             <option value="REJECTED">Rejected</option>
+            <option value="BVRC999">Request could not be processed by recipient (Alias: Sygna BVRC Rejected Code)</option>
```
Comment (bbengfort) [inline pkg/web/templates/components/transaction_rejection.html:17]:
```suggestion
```

I'm not sure Sygna is still around -- but Envoy customers probably won't be using Sygna so I don't think we have to expose the Sygna codes to them.

**Review by bbengfort — APPROVED** [review]
Thanks for making that fix and for your attention to detail with the API docs!

**Pull request merged**